lib/Controller/Campaign.php

```diff
@@ -68,17 +74,19 @@ class Campaign extends Base
      * @param ConfigServiceInterface $config
      * @param CampaignFactory $campaignFactory
      * @param LayoutFactory $layoutFactory
+     * @param TagFactory $tagFactory
```

Comment:
interesting positioning of this param comment - perhaps it should be at the end? :)